news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc

```diff
@@ -0,0 +1,62 @@
+= Proposed Final Draft of Bean Validation 2.0 released
+Gunnar Morling
+:awestruct-layout: news
+:awestruct-tags: [ "release" ]
+
+I'm very happy to announce that Bean Validation 2.0 has published its Proposed Final Draft (CR 1)!
+
+You can find the PFD right here on link:/2.0/spec/2.0.0.cr1/[beanvalidation.org] as well as on JSR 380's pages on https://jcp.org/aboutJava/communityprocess/pfd/jsr380/index.html[jcp.org].
+We've also prepared a link:/2.0/spec/2.0.0.cr1/diff/diff-to-1.1/[colored diff] with all the changes since Bean Validation 1.1.
+The updated API is available on Maven Central, using the GAV coordinates are _javax.validation:validation-api:2.0.0.CR1_.
+Alternatively, it's part of the ZIP that can be downloaded from https://jcp.org/aboutJava/communityprocess/pr/jsr380/index.html[from jcp.org].
+
+== What's new in Bean Validation 2.0?
+
+The focus of Bean Validation 2.0 is supporting and taking advantage of Java SE 8.
+
+For instance type annotations are used to validate the elements of generic containers: `List<@NotNull @Email String> emails`.
+There are new built-in constraints (e.g. `@NotBlank`, `@NotEmpty`, `@Email`, `@Positive` and `@Negative`) and all built-in constraints are repeatable annotations now.
+Bean Validation 2.0 also supports the new Java 8 date and time types (JSR 310), the property types defined by JavaFX (`StringProperty` etc.) as well as `java.util.Optional`.
+
+To learn more about all the new functionality in Bean Validation 2.0,
```

Comment:
I'll change it to "features", although the second definition in your link fits very well:

>  the set of functions or capabilities associated with computer software

Don't you think so?